metis-indexing/src/test/java/eu/europeana/indexing/tiers/view/ContentTierBreakdownTest.java

```diff
@@ -67,4 +67,4 @@ void objectCreationTest_Without_Processing_Errors_Parameter() {
         contentTierBreakdown.getMediaResourceTechnicalMetadataList()));
 
   }
-}
\ No newline at end of file
+}
```

Comment:
A test is missing with ```true``` value to cover the new feature.

Reply:
Fixed it